cli/docs/flags.go

```diff
@@ -117,6 +118,9 @@ var commandFlags = map[string][]string{
 		url, user, password, accessToken, ServerId, SpecFlag, Threads, scanRecursive, scanRegexp, scanAnt,
 		Project, Watches, RepoPath, Licenses, OutputFormat, Fail, ExtendedTable, BypassArchiveLimits, MinSeverity, FixableOnly,
 	},
+	Enrich: {
+		url, user, password, accessToken, ServerId, SpecFlag, Threads,
+	},
```

Comment:
```suggestion
	Enrich: {
		url, user, password, accessToken, ServerId, SpecFlag, Threads, Fail, Project, Watches, RepoPath,
	},
```
I see that you are expecting `fail` option but you did not add it as a flag for the command and parsed it's value to the cmd.
In addition, this is relevant only when we have context to the scan (watch, project key, repoPath...) none of them are supported flags in your command.

Reply:
removed fail option